x-pack/plugins/cases/public/components/create/index.tsx

```diff
@@ -17,6 +17,7 @@ import { SubmitCaseButton } from './submit_button';
 import { Case } from '../../containers/types';
 import { CaseType } from '../../../common';
 import { CasesTimelineIntegration, CasesTimelineIntegrationProvider } from '../timeline_context';
+// import { CasesLensIntegration, CasesLensIntegrationProvider } from '../lens_context';
```

Comment:
will add it in the follow up PR

Reply:
can we put it behind a feature flag instead?

Reply:
I like the idea to  add a Feature flag just in case we find any odd behavior.